v2/bundle/spiffebundle/bundle.go

```diff
@@ -0,0 +1,142 @@
+package spiffebundle
+
+import (
+	"crypto"
+	"crypto/x509"
+	"io"
+	"time"
+
+	"github.com/spiffe/go-spiffe/v2/bundle/jwtbundle"
+	"github.com/spiffe/go-spiffe/v2/bundle/x509bundle"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+// Bundle is a collection of trusted public key material for a trust domain.
+type Bundle struct {
+}
+
+// New creates a new bundle
+func New(trustDomain spiffeid.TrustDomain) *Bundle {
+	panic("not implemented")
+}
+
+// Load loads a Bundle from a file on disk.
+func Load(trustDomain spiffeid.TrustDomain, path string) (*Bundle, error) {
+	panic("not implemented")
+}
+
+// Read decodes a bundle from a reader.
+func Read(trustDomain spiffeid.TrustDomain, r io.Reader) (*Bundle, error) {
+	panic("not implemented")
+}
+
+// Parse parses a bundle from bytes.
+func Parse(trustDomain spiffeid.TrustDomain, b []byte) (*Bundle, error) {
+	panic("not implemented")
+}
+
+// FromX509Bundle creates a Bundle from an X.509 bundle.
+func FromX509Bundle(x509Bundle *x509bundle.Bundle) *Bundle {
+	panic("not implemented")
+}
+
+// FromJWTBundle creates a Bundle from a JWT bundle.
+func FromJWTBundle(jwtBundle *jwtbundle.Bundle) *Bundle {
+	panic("not implemented")
+}
+
+// TrustDomain returns the trust domain of the bundle.
+func (b *Bundle) TrustDomain() spiffeid.TrustDomain {
+	panic("not implemented")
+}
+
+// X509Roots returns the X.509 roots in the bundle.
+func (b *Bundle) X509Roots() []*x509.Certificate {
+	panic("not implemented")
+}
+
+// JWTKeys returns the JWT keys in the bundle.
+func (b *Bundle) JWTKeys() map[string]crypto.PublicKey {
```

Comment:
what are the keys in the map ? can you clarify it in the comment ?

Reply:
Done